authentication/models/user.py

```diff
@@ -37,48 +39,37 @@ def _create_user(self, id, first_name, last_name, email, password, birthday, pro
 
 
 class User(AbstractBaseUser):
-    id = models.CharField(primary_key=True, max_length=30, verbose_name="User id")
-    first_name = models.CharField(max_length=50, verbose_name="User first name")
-    last_name = models.CharField(max_length=50, verbose_name="User last name")
-    promo = models.IntegerField(verbose_name="Promotion")
-
-    email = models.EmailField(
-        verbose_name='email address',
-        max_length=160,
-        unique=True,
-    )
+    id = models.CharField(primary_key=True, max_length=30)
 
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
     nickname = models.CharField(max_length=128, blank=True, default="")
-    birthday = models.DateField(null=True, verbose_name="date de naissance")
-
-    # Contact the person
-    phone = models.CharField(max_length=15, blank=True, verbose_name="numéro de téléphone")
+    birthday = models.DateField(null=True)
+    email = models.EmailField(max_length=160, unique=True)
+    promo = models.IntegerField(verbose_name="Promotion",
```

Comment:
do we leave `verbose_name` ?

Reply:
I left it because “promo” is not an English word